Dockerfile

```diff
@@ -38,22 +33,19 @@ RUN yum update -y && yum install -y deltarpm && \
                     java-11-openjdk \
                     java-11-openjdk-devel \
                     lbzip2 \
-                    lbzip2-utils \
                     libffi \
                     libffi-devel \
                     libX11-devel \
                     libxml2-devel \
                     libxslt-devel \
-                    lockfile-progs \
                     make \
                     net-tools \
                     openldap-devel \
                     openssh-server \
                     openssl-devel \
                     perl \
                     procmail \
-                    python-devel \
```

Comment:
This seems to have been renamed to Python3-devel, which es most probably exactly what we want.